tools/load-test/utils.go

```diff
@@ -0,0 +1,283 @@
+						return
+					}
+					log.Printf("Name of the failed pod is: %v", string(functionPodName))
+					log.Fatalf("Response is not equal to expected output:\nResponse: %v\nExpected: %v", string(bodyBytes), expectedOutput)
+				} else {
+					log.Printf("Response from function: %v", resp.StatusCode)
+					bodyBytes, err := ioutil.ReadAll(resp.Body)
+					if err != nil {
+						log.Printf("Unable to get response: %v", err)
+					}
+					log.Printf("Response body is: %v", string(bodyBytes))
+				}
+			}
+		}
+	}
+}
+
+func deployK8s(yamlFile string) (string, error) {
+	cmd := exec.Command("kubectl", "create", "-f", yamlFile, "-n", namespace)
+	stdoutStderr, err := cmd.CombinedOutput()
+	output := string(stdoutStderr)
+	return output, err
+}
+
+const lettersAndNums = "abcdefghijklmnopqrstuvwxyz0123456789"
+
+func randomString(n int) string {
+	b := make([]byte, n)
+	for i := range b {
+		b[i] = lettersAndNums[rand.Intn(len(lettersAndNums))]
+	}
+	return string(b)
+}
+
+func getHTTPClient(skipVerify bool) *http.Client {
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: skipVerify},
+	}
+	client := &http.Client{Transport: tr}
+	return client
+}
+
+func cleanup() {
+	log.Println("Cleaning up resources")
+	deleteFun()
+	deleteNamespace()
+}
+
+func deleteFun() {
+
+	timeoutDelFunc := time.After(15 * time.Minute)
+	tick := time.Tick(1 * time.Second)
+	done := false
+	for {
+		var err error
+		select {
+		case <-timeoutDelFunc:
+			log.Fatalf("Unable to delete function: %s error: %v \n", functionName, err)
+		case <-tick:
+			_, err = deleteK8s(functionYaml)
+			if err == nil {
+				log.Println("Execution of delete function is successful!")
+				done = true
+			}
+		}
+		if done {
+			break
+		}
+	}
+
+	for {
+		select {
+		case <-timeoutDelFunc:
+			log.Fatal("Timed out waiting for ", functionName, " pod to be deleted\n")
+		case <-tick:
+			cmd := exec.Command("kubectl", "-n", namespace, "get", "pod", "-l", "function="+functionName)
+			stdoutStderr, err := cmd.CombinedOutput()
+			if err == nil && strings.Contains(string(stdoutStderr), "No resources found") {
+				log.Printf("All functions cleaned up!")
+				return
+			}
+		}
+	}
+}
+
+func deleteNamespace() {
+	stdoutStderr, err := deleteK8s(nsYaml)
+	output := string(stdoutStderr)
+	if err != nil && !strings.Contains(output, "not found") && !strings.Contains(output, "The system is ensuring all content is removed from this namespace") {
+		log.Fatal("Unable to delete namespace ", namespace, ":\n", output)
+	}
+	timeoutNSDelete := time.After(20 * time.Minute)
+	tick := time.Tick(1 * time.Second)
+	for {
+		cmd := exec.Command("kubectl", "get", "ns", namespace, "-oyaml")
+		select {
+		case <-timeoutNSDelete:
+			cmd = exec.Command("kubectl", "get", "po,svc,deploy,function,rs,hpa", "-n", namespace, "-oyaml")
+			stdoutStderr, err := cmd.CombinedOutput()
+			if err != nil {
+				log.Fatalf("Unable to get ns: %v\n", string(stdoutStderr))
```

Comment:
may be `Unable to get objects in namespace`

Reply:
fixed